apps/web-giddh/src/app/purchase/purchase-setting/purchase-setting.component.ts

```diff
@@ -72,6 +75,10 @@ export class PurchaseSettingComponent implements OnInit, OnDestroy {
 
         this.store.dispatch(this.settingsIntegrationActions.GetGmailIntegrationStatus());
         this.initSettings();
+        this.invoiceSettings.purchaseBillSettings.sendThroughGmail = false;
```

Comment:
Can we assign these default values at line https://github.com/Walkover-Web-Solution/Giddh-New-Angular4-App/blob/3a08674c9ccb75377f2dafba692a694f23730db1/apps/web-giddh/src/app/purchase/purchase-setting/purchase-setting.component.ts#L25